doc/conf_ref.rst

```diff
@@ -870,6 +870,8 @@ configuration.
 
     Exclude packages of this repository, specified by a name or a glob and
     separated by a comma, from all operations.
+    Do not add multiple excludepkgs lines in the configuration file,
+    because yum only consider the last excludepkgs entry.
```

Comment:
Could you rephrase the second line to:
```
Do not add multiple excludepkgs lines in the configuration file,
because only the last excludepkgs entry is considered.
```

Reply:
Thanks for the comment.
I have modified the 2nd line.

Reply:
Unfortunately that is even worse, the second line is not very understandable.

You could just use:
```
Do not add multiple excludepkgs lines in the configuration file
because only the last excludepkgs entry is considered.
```
(I have also removed the `,` which is redundant.